packages/vite/src/node/ssr/ssrExternal.ts

```diff
@@ -24,22 +22,50 @@ export function resolveSSRExternal(
   ssrExternals: Set<string> = new Set(),
   seen: Set<string> = new Set()
```

Comment:
We don't recurse `resolveSSRExternal()` anymore so we can remove `ssrExternals` and `seen` from the argument list and define them with `const` instead.

```js
const ssrExternals = new Set()
const seen = new Set() 
```

Reply:
good suggestion. updated!